docs/UserGuide.md

```diff
@@ -99,6 +99,10 @@ startup investments!
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* Currently selected startup card will appear blue!
```

Comment:
Would it be good for some image or graphics to come together with this? I'm worried that solely saying "in blue" might be confusing for users, but I'm not too sure either.